app/controllers/comments_controller.rb

```diff
@@ -1,13 +1,14 @@
 # frozen_string_literal: true
 
 class CommentsController < ApplicationController
+  before_action :set_giveaway
```

Comment:
Similar to **:find_comment**, you can specify which actions is this method set for.